db/migrate/20170901005004_usage_reports.rb

```diff
@@ -0,0 +1,17 @@
+class UsageReports < ActiveRecord::Migration
+  def change
+    create_table :usage_reports do |t|
+      t.integer :groups_count
+      t.integer :users_count
+      t.integer :discussions_count
+      t.integer :polls_count
+      t.integer :comments_count
+      t.integer :stances_count
+      t.integer :reactions_count
+      t.integer :visits_count
+      t.string  :canonical_host
```

Comment:
I think we're likely going to want to index on canonical_host (so that we can search by host name), or alternatively assign a site_id integer to each unique canonical host for easy lookup / graphs over time etc.